aspnetcore/razor-pages/rp-mvc.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Razor Pages compared to ASP.NET Core MVC with controllers and views
+author: Rick-Anderson
+description: Contrast and compare Razor Pages to controller/view programming in ASP.NET Core
+monikerRange: '>= aspnetcore-2.0'
+ms.author: riande
+ms.date: 08/05/2018
+uid: razor-pages/rp-mvc
+---
+# Razor Pages compared to ASP.NET Core MVC
+
+By [Scott Sauber](https://twitter.com/scottsauber) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+Razor Pages is a feature of the ASP.NET Core MVC framework. Razor Pages follows the [MVVM](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/enterprise-application-patterns/mvvm) pattern. Developers and teams familiar with ASP.NET MVC development:
```

Comment:
To me this kinda reads a little funny that it says it's a feature of the MVC framework, then immediately says it follows MVVM which someone might be like "what?  how can it be a feature of MVC if it follows a (slightly) different pattern?"  Just kinda the unfortunate naming of MVC than anything else though.